implementation/operations.go

```diff
@@ -0,0 +1,144 @@
+package implementation
+
+import (
+	"errors"
+	"github.com/aditya/ProjectCatalog/models"
+	"sort"
+	"strconv"
+	"time"
+)
+
+type Inmemoryimplement struct{
+	ProductArr []models.Product
+
+}
+
+type TimeStampStructure struct{
+	prodId int64
+	timeInSec int64
+}
+
+var mp map[TimeStampStructure]int64
+
+func Initializer(){
```

Comment:
also, initialize `Inmemoryimplement ` and add a getter for the same in this package